tests/unit/plugins/modules/database/saphana/test_hana_query.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber (@rainerleber) <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.community.general.tests.unit.compat import unittest
+from ansible_collections.community.general.tests.unit.compat.mock import patch
+from ansible_collections.community.general.plugins.modules import hana_query
+from ansible_collections.community.general.tests.unit.plugins.modules.utils import AnsibleExitJson, AnsibleFailJson, set_module_args
+from ansible.module_utils import basic
+from ansible.module_utils.common.text.converters import to_bytes
+import json
+
+
+def set_module_args(args):
```

Comment:
There's no need to copy these. Simply import them from `ansible_collections.community.general.tests.unit.plugins.modules.utils`.